packages/fast-components-class-name-contracts-base/src/button/index.ts

```diff
@@ -2,5 +2,8 @@
  * The class name contract for the button component
  */
 export interface IButtonClassNameContract {
+    /**
+     * The root button component

```

Comment:
This reads a bit weird to me. This is technically the root element of the component. Can we consider an update to make these more clear?